app/models/Backend.scala

```diff
@@ -786,6 +788,8 @@ class Backend @Inject() (implicit
     val indexTable = defaultOTSettings.clickhouse.literatureIndex
     logger.info(s"query literature ocurrences in table ${table.name}")
 
+    println("cursor" + cursor)
```

Comment:
We should probably remove this debug message

Reply:
applied